docs/specification.md

```diff
@@ -442,13 +445,15 @@ Updates or creates a new [position](#positions) in the given range.
   fails with `high_tokens_err` error code.
 - A user can set `delta_liquidity` to `0` on an existing position to simply retrieve
   any uncollected fees.
+- This fails with `price_out_of_bounds_err` when a tick index out of `[-1048575; 1048575]`
+  range is provided.
 
 ```ocaml
 type set_position_param = {
-    i_l : tick_index ;
-    i_u : tick_index ;
-    i_l_l : tick_index ;
-    i_u_l : tick_index ;
+    lower_tick_index : tick_index ;
+    upper_tick_index : tick_index ;
+    lower_tick_witness : tick_index ;
+    upper_tick_witness : tick_index ;
```

Comment:
I think the field names in `sci_param` (in the `snapshot_cumulatives_inside` section) should probably also be updated.

Reply:
I knew you will raise this concern :smile: 

This will be fixed in #24 [here](https://github.com/serokell/segmented-cfmm/pull/24/files#diff-eebad86dc55e949d1d0630c4a2e92fed2a16b19e0ab623719ed429b5d3eee101R488).